.github/workflows/deploy-to-control-plane-review.yml

```
@@ -1,85 +1,174 @@
-            PR_NUMBER=$(echo "$API_RESPONSE" | jq '.[] | select(.head.ref=="'$REF'") | .number')
-          fi
-          echo "PR_NUMBER: $PR_NUMBER"
-          if [ -z "$PR_NUMBER" ]; then
-            echo "PR_NUMBER is not set. Aborting."
-            exit 1
-          fi
-          echo "PR_NUMBER=$PR_NUMBER" >> $GITHUB_ENV
-      - name: Get App Name
+      - name: Configure app name
+        id: app-config
         run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:2:30: Double quote to prevent globbing and word splitting

(shellcheck)